plugins/git/octo.nix

```diff
@@ -0,0 +1,103 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "octo";
+  originalName = "octo.nvim";
+  defaultPackage = pkgs.vimPlugins.octo-nvim;
+
+  maintainers = [ helpers.maintainers.svl ];
+
+  settingsOptions = {
+
+    use_local_fs = helpers.defaultNullOpts.mkBool false "Use local files on right side of reviews.";
+
+    enable_builtin = helpers.defaultNullOpts.mkBool false "Shows a list of builtin actions when no action is provided.";
+
+    default_remote = helpers.defaultNullOpts.mkListOf types.str [
+      "upstream"
+      "origin"
+    ] "Shows a list of builtin actions when no action is provided.";
```

Comment:
I am surprised that it works to supply the plugin default as a list rather than as a string (`''["upstream" "origin"]''`).

Reply:
works. i checked the lua config and it translates correctly
```lua
["default_remote"] = {"upstream","yourmum"}
```

not relevant, but how do you get the init.lua path? nix build doesn't have it. I currently add something to extraConfigLua for formatter to fail and it prints the store path

Reply:
> works. i checked the lua config and it translates correctly

The value itself sure, but the default you provide as the second argument of `mkListOf` is a **string** that will be injected in the option documentation.
Here you give a list.

> not relevant, but how do you get the init.lua path? nix build doesn't have it. I currently add something to extraConfigLua for formatter to fail and it prints the store path

We provide a little helper that you can run: `nixvim-print-init`.

Reply:
the difference is `"upstream" "origin"` without quoting and  `[ "upstream" "origin" ]` with
i added the quotes